shared/utils/yaml-to-shorthand.py

```diff
@@ -226,6 +226,7 @@ def __init__(self, id_):
         self.front_matter = ""
         self.rear_matter = ""
         self.cpes = []
+        self.platform_cpes = []
```

Comment:
CPEs and platform CPEs? IMO we should take this opportunity to unify it. There are only one CPEs. Calling it platform may also be misleading because we use them for JRE, Firefox, etc... You can think of apps as platforms but IMO it's a bit misleading.

Reply:
OK, what does this mean in practice? I guess that we are still going to distinguish between them in the code (as one set of CPEs comes from benchmark config and the other fone from the product config), but they should both be `cpes:` in yamls?

Reply:
What's the difference between platform_cpes and cpes? Do we need both? Probably not because the confusion is not worth the benefit if there even is any benefit..